mne/commands/mne_setup_source_space.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+"""Set up bilateral hemisphere surface-based source space with subsampling.
+
+Examples
+--------
+.. code-block:: console
+
+    $ mne setup_source_space --subject sample
+
+
+ .. note : Only one of --ico, --oct or --spacing options can be set at the same
+           time. Default to spacing = 7mm.
+
+"""
+
+import sys
+
+import mne
+
+
+def run():
+    """Run command."""
+    from mne.commands.utils import get_optparser
+
+    parser = get_optparser(__file__)
+
+    parser.add_option('-s', '--subject',
+                      dest='subject',
+                      help='Subject name (required)',
+                      default=None)
+    parser.add_option('--src', dest='fname',
+                      help='Output file name. Use a name <dir>/<name>-src.fif',
+                      metavar='FILE', default=None)
+    parser.add_option('--morph',
+                      dest='subject_to',
+                      help='morph the source space to this subject',
+                      default=None)
+    parser.add_option('--surf',
+                      dest='surface',
+                      help='The surface to use. (default to white)',
+                      default='white',
+                      type='string')
+    parser.add_option('--spacing',
+                      dest='spacing',
+                      help='Specifies the approximate grid spacing of the '
+                           'source space in mm. (default to 7mm)',
+                      default=None,
+                      type='int')
+    parser.add_option('--ico',
+                      dest='ico',
+                      help='use the recursively subdivided icosahedron '
+                           'to create the source space.',
+                      default=None,
+                      type='int')
+    parser.add_option('--oct',
+                      dest='oct',
+                      help='use the recursively subdivided octahedron '
+                           'to create the source space.',
+                      default=None,
+                      type='int')
+    parser.add_option('-d', '--subjects-dir',
+                      dest='subjects_dir',
+                      help='Subjects directory',
+                      default=None)
+    parser.add_option('--cps',
+                      dest='cps',
+                      help='Add patch information to source space.',
+                      default=True)
```

Comment:
I don't think the C command defaults to true. It is only used if you pass --cps argument.

Reply:
But in Python it does default to `True` (as does `add_dist`) and with good reason, that the normals are better (and later inter-source distances are available). So I'd rather stay consistent internally (use True) than move to what MNE-C does (use False)

Reply:
If we use default to True, we should be more clear and use a flag such as  "nocps"